plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -17,71 +16,15 @@ import (
 	"strings"
 )
 
-// mask for masking username/password from error messages
-var mask = regexp.MustCompile(`https?:\/\/\S+:\S+@`)
```

Comment:
Searching under Telegraf, I did not see anything using it, nor did I see something similar in other .go files.  Did I miss something? I can certainly restore it if you'd like - let me know.